_data/import-public.rb

```diff
@@ -17,3 +19,17 @@
     end
   end
 end
+
+data = JSON.load File.read(File.join(DATA_DIR, 'team.json'))
+fm_rexexp = Regexp.new('^---.*---$', Regexp::MULTILINE)
+
+Dir[File.join [DATA_DIR] + %w(.. _team *.md)].each do |member_file|
+  name = File.basename(member_file)[0..-4] # Trim .md extension
```

Comment:
Yep! Though, splitting on `.` alone may prove reliable. In fact, just noticed that `File.basename` takes a suffix argument that does the job. Updated.

Also, FYI, `Jekyll::YAML_FRONT_MATTER_REGEXP` is defined in `lib/jekyll/document.rb` and duplicated in `Jekyll::Convertible::read_yaml` (`lib/jekyll/convertible.rb`). I'll update this to use that define regexp.